srv/GetSimulatorFeatures.srv

```diff
@@ -0,0 +1,32 @@
+# Get a list of features supported by the simulator.
+# Simulators adhering to the standard need to implement at least this interface.
+
+---
+
```

Comment:
Would you consider spinning this out to its own message, SimulatorFeatures.msg, separate from the service? I'm not sure what the motivation is to make this a service rather than a "latched" topic.

Reply:
What would you point to as advantages of a latched topic over a service for this message in particular?

Reply:
It's a minor point, but having this as a message would mean that you could have all these constants in a message OR a service, so that in the future, if someone decides that they want it as a message, they can have it without having to duplicate or refactor.